File: gamehub-android/game/src/main/java/com/farsitel/bazaar/game/data/Event.java
package com.farsitel.bazaar.game.data;

public class Event {
    String id, startTimestamp, endTimestamp;

    public Event(String id, String startTimestamp, String endTimestamp) {
        this.id = id;
        this.startTimestamp = startTimestamp;
        this.endTimestamp = endTimestamp;
    }
}

File: gamehub-unity/Assets/Bazaar/GameHub/Scripts/Data/Event.cs
using UnityEngine;

namespace Bazaar.GameHub.Data
{
    public class Event
    {
        public string id;
        public string startTimestamp;
        public string endTimestamp;
        
        public Event(AndroidJavaObject entity)
        {
            this.id = entity.Get<string>("id");
            this.startTimestamp = entity.Get<string>("startTimestamp");
            this.endTimestamp = entity.Get<string>("endTimestamp");
        }

        override public string ToString() => $"id: {id}, startTimestamp: {startTimestamp}, endTimestamp: {endTimestamp}";
    }
}